examples/vault-auto-unseal/README.md

```diff
@@ -49,7 +49,7 @@ even if you immediately delete it.
 1. Ssh to an instance in the vault cluster and run `vault operator init` to initialize
   the cluster, then `vault status` to check that it is unsealed. If you ssh to a
   different node in the cluster, you might have to restart Vault first with
-  `sudo supervisorctl restart vault` so it will rejoin the cluster and unseal.
+  `sudo systemctl restart vault` so it will rejoin the cluster and unseal.
```

Comment:
Would it be `restart vault` or `restart vault.service`?

Reply:
Oops, I'll change it to `vault.service` for consistency.